### PR TITLE
Created Contact-us page content

### LINKED ISSUE #10: Frontend: Contact us
## User story

**Who:** **As a** developer
**What:** **I want to** be able to display the Contact Page
**Why:** **so that I can** show that info to the users.

## Acceptance criteria

Create a React component like the one we can see below
 
Screen Shot 2022-01-20 at 11 13 35

Check the wireframes here: https://www.figma.com/file/yo4QT2EcPo83HAMI3ydMxb/HYF-Wireframes?node-id=4%3A1324

## Implementation details

Download the images from the wireframes and add them to this folder src/client/assets/images

Add a folder called fonts under assets - /src/client/assets/ and add the font files.

### PR DESCRIPTION
**Pull request by Divyajg**
# Description

Created Contact-us page content with some  links:(mailto and tel)

Fixes # 10

# How to test?

Test using npm start.

# Checklist

 -[X] I have performed a self-review of my own code
 -[X] I have followed the name conventions for CSS Classnames and filenames, Components names and filenames, Style filenames, here https://github.com/HackYourFuture-CPH/curriculum/blob/master/review/review-checklist.md
 -[X] I have commented on my code, particularly in hard-to-understand areas, if your code was simple enough mark the box anyway
 -[X] I have made corresponding changes to the documentation, if your code was simple enough mark the box anyway
 -[X] This PR is ready to be merged and not breaking any other functionality

#10-contact-us-page

### CODE REVIEW

**Comment by h09shais** [comment]
Hi @Divyajg !
What is the reason to delete `.env.example` 😀 ?

**Comment by Divyajg** [comment]
I have just changed the same .env.example to .env file with the relevent data.
Should I revert that?

File: src/client/components/contact-page/contact.css [hunk 1]
```diff
@@ -0,0 +1,53 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
+
+.page-margin {
```
Comment (senner007) [inline src/client/components/contact-page/contact.css]: The style names have to be more specific here. If not, the css will compete with other stylesheets.

Another approach is to use css modules to encapsulate the css, like this:
https://www.w3schools.com/react/react_css.asp

Still, its best practice to create names that describe very precisely what they contain.

File: src/client/components/contact-page/Contact.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import image from '../../assets/images/image26.png';
+import './contact.css';
+
+export default function ContactPage() {
```
Comment (senner007) [inline src/client/components/contact-page/Contact.js:5]: It is preferrable to create components that are only concerned with one thing. Its _probably_ not going to be critical here, but once the code base starts to grow, we need to start reusing components, make changes and move things around.

It is demonstrated here:
https://reactjs.org/docs/thinking-in-react.html

**Review by senner007 — CHANGES_REQUESTED** [review]

File: src/client/components/contact-page/Contact.js (hunk 2 above)
Comment (Divyajg) [reply src/client/components/contact-page/Contact.js:5]: Yes, I understand and I note that.

**Review by JiuTak — APPROVED** [review]
look good on my local and shows the same as Figma :)
becare of the merge conflict btw
_the image is already on the develop branch I pushed and it is the same name_

**Review by suvarna-ratna — APPROVED** [review]
Working as it should ... Pls update path in header and footer also like /contact-page

**Review by FlorentinaPetica — APPROVED** [review]
It looks ready for merging. 👍

**Pull request merged**